<!-- Launch a basic bump bot first test with the Kinect mounted on the Create -->
<launch>
  <param name="robot_description" command="$(find xacro)/xacro.py '$(find create_urdf)/urdf/create.urdf.xacro'" />

  <node pkg="robot_state_publisher" type="state_publisher" name="robot_state_publisher" output="screen">
    <param name="publish_frequency" type="double" value="30.0" />
    <!--param name="tf_prefix" type="string" value="" /-->
  </node>

  <!-- Create Driver -->
  <node pkg="create_node" type="create_node.py" name="create_node" >
    <param name="bonus" value="false" />
    <param name="~port" value="/dev/ttyUSB0" />
    <param name="~drive_mode" value="twist" />
    <param name="~has_gyro" value="false" />
  </node>

  <!-- Launch the first test program -->
  <node pkg="create_kinect" type="first_test" name="first_test">
    <param name="enable_create" value="true"/>
    <param name="theta" value="-0.25"/>
    <param name="xl_min" value="-1.0"/>
    <param name="xl_max" value="0.0"/>
    <param name="xr_min" value="0.0"/>
    <param name="xr_max" value="1.0"/>
    <param name="y_min" value="-0.1"/>
    <param name="y_max" value="0.4"/>
    <param name="z_min" value="0.25"/>
    <param name="z_max" value="1.0"/>
  </node>
</launch>
